DatadogLogs/Sources/Feature/MessageReceivers.swift

```diff
@@ -189,7 +189,7 @@ internal struct CrashLogReceiver: FeatureMessageReceiver {
 
         // crash reporting is considering the user consent from previous session, if an event reached
         // the message bus it means that consent was granted and we can safely bypass current consent.
-        core.v1.scope(for: DatadogLogsFeature.name)?.eventWriteContext(bypassConsent: true, forceNewBatch: false) { _, writer in
+        core.scope(for: DatadogLogsFeature.name)?.eventWriteContext(bypassConsent: true, forceNewBatch: false) { _, writer in
```

Comment:
This was a miss! It didn't change the behaviour since `DatadogV1CoreProtocol` was extending the `DatadogCoreProtocol`.